mage_ai/data_preparation/models/block/__init__.py

```diff
@@ -1859,3 +1921,67 @@ def execute_block_function(
                 print('Sensor sleeping for 1 minute...')
                 time.sleep(60)
             return []
+
+
+class CallbackBlock(Block):
```

Comment:
It'll be bettter to put this class in a separate file.

Reply:
ok i can move it out